src/js/components/HealthBar.js

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import React from 'react';
+import {Tooltip} from 'reactjs-components';
+
+import HealthBarMap from '../constants/HealthBar';
+import StatusBar from './StatusBar';
+
+class HealthBar extends React.Component {
+  renderToolTip() {
+    let {tasks, instancesCount} = this.props;
+
+    tasks = Object.keys(tasks).filter(function (task) {
+      return tasks[task] !== 0 && task !== 'tasksRunning';
+    }).map(function (task, index) {
+      let percentage = parseInt(tasks[task] / instancesCount * 100, 10);
+
+      let classSet = classNames(HealthBarMap[task].cssName, 'dot icon');
+
+      return (
+        <div key={index}>
+          <span className={classSet}></span>
+          {` ${tasks[task]} ${HealthBarMap[task].label} (${percentage} %)`}
+        </div>
+      );
+    }).filter(function (item) {
+      return item != null;
+    });
+
+    if (tasks.length === 0) {
+      return 'No Running Tasks';
+    }
+
+    return tasks;
+  }
+
+  render() {
+    let {tasks, instancesCount} = this.props;
+
+    if (tasks == null) {
+      return null;
+    }
+
+    tasks = Object.keys(tasks).filter(function (task) {
```

Comment:
+1
One thing we realized about two months ago was to use unit tests to test methods in a component and leave the testing of the rendering to integration tests.
Putting it into a method allows us to test the unit only.
Then the integration tests don't have to care if the components are called Tooltip/StatusBar, it's just about what's rendered :)
...sorry for the verbose explanation.